cmd/system-probe/app/run.go

```diff
@@ -0,0 +1,222 @@
+package app
```

Comment:
How much of this file is existing code?

Reply:
It is mostly moved from the previous `main.go`, but they are small differences to handle the different control flow.